pom.xml

```diff
@@ -77,6 +113,17 @@
 
     <build>
         <plugins>
+            <!--Deployment / build plugins-->
+            <plugin>
+                <groupId>org.sonatype.plugins</groupId>
+                <artifactId>nexus-staging-maven-plugin</artifactId>
+                <version>1.6.5</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <!--Ref.: http://books.sonatype.com/nexus-book/reference/staging-deployment.html-->
```

Comment:
@ahaarrestad I have seen that there was a blank between <!-- and the comment message in the other comments. Maybe this was the confusion. Added the missing blank.